lib/watcher.js

```diff
@@ -44,12 +44,10 @@ function Watcher(logger, api, files, sources) {
 
 	this.isTest = makeTestMatcher(files, AvaFiles.defaultExcludePatterns());
 
-	var isFirstRun = true;
 	this.clearLogOnNextRun = true;
+	this.runVector = 0;
 	this.run = function (specificFiles) {
-		if (isFirstRun) {
-			isFirstRun = false;
-		} else {
+		if (this.runVector > 0) {
```

Comment:
I'm trying to avoid implicit conversions. The vector is increasing so `> 0` signifies that more than `!== 0` would, but not fussed either way.